users/managers.py

```diff
@@ -1,11 +1,14 @@
 """
 Contains Managers for models
 """
+# pylint: disable= no-member, cyclic-import
 from django.contrib.auth.base_user import BaseUserManager
 from django.contrib.auth.password_validation import validate_password
 from django.core.validators import validate_email
 from django.utils.translation import ugettext_lazy as _
 
+import users.models
```

Comment:
For better readability and usages you can do something like below. That way you won't have to write `users.models.Role` or `users.models.User`
```suggestion
from users.models import User, Role
```

Reply:
This causes a `circular import error` as we are importing `managers.py` in `models.py`